qiskit/ignis/verification/tomography/basis/circuits.py

```diff
@@ -362,8 +371,18 @@ def _tomography_circuits(circuit, measured_qubits, prepared_qubits=None,
 # Built-in circuit functions
 ###########################################################################
 
-def default_basis(basis):
-    """Built in Tomography Bases."""
+def default_basis(basis: Union[str, TomographyBasis]) -> TomographyBasis:
+    """Built in Tomography Bases.
+
+    Args:
+        basis: Either a **TomographyBasis** element
```

Comment:
This sentence might be better moved out of Args as separate documentation.

```
If the input basis is ``None`` or a :class:`TomographyBasis` it will be returned unchanged.
If it is a `"Pauli"` or `"SIC"` it will return the built in tomography basis object for that basis.

Args:
    basis: the tomography basis.
```